go/vt/vttablet/onlineddl/executor.go

```diff
@@ -5355,3 +5355,15 @@ func (e *Executor) OnSchemaMigrationStatus(ctx context.Context,
 
 	return e.onSchemaMigrationStatus(ctx, uuidParam, status, dryRun, progressPct, etaSeconds, rowsCopied, hint)
 }
+
+func (e *Executor) checkOnPreparedPool(table string, waitTime time.Duration, retries int) error {
```

Comment:
Let's please rewrite this function to accept a `ctx` and to check on `ctx.Done`, even though this function is only called with `100*time.Miliseconds` which is very short.

Ideally let's run a classic `time.Ticker` loop instead of `time.Sleep()`. Or at the very least 
```go
select {
  case <- time.After(...):
  case <- ctx.Done():
    return _some error_
}
```